app/calculator.py

```diff
@@ -0,0 +1,16 @@
+import streamlit as st
+
+def calculate(choice, num_1: float, num_2:float):
+    if choice == "Addition":
```

Comment:
Add docstring for the function is a good practice for you and other programmers to understand better the functions. The docstring should consist of 
- what the function is do
- what is the parameter means
- what the function is return